Resources/public/js/views/ez-navigationhubview.js

```diff
@@ -130,7 +130,10 @@ YUI.add('ez-navigationhubview', function (Y) {
                     activeZone = key;
                 }
             }, this);
-            this.set('activeNavigation', activeZone);
+
+            if (this.get('activeNavigation') !== null) {
```

Comment:
I think this is the wrong place for this condition. Here the `activeNavigation` attribute should always be set. But in the `activeNavigationChange` handler in the NavigationHubViewService, you should check the previous `activeNavigation` value to trigger the first route navigation only if there was a `activeNavigation` before, meaning the user has chosen a zone.